x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/all/columns.tsx

```diff
@@ -93,10 +89,10 @@ export const getActions = (
     ) : (
       i18n.DUPLICATE_RULE
     ),
-    enabled: (rowItem: TableItem) => canEditRuleWithActions(rowItem, actionsPrivileges),
-    onClick: async (rowItem: TableItem) => {
+    enabled: (rowItem: Rule) => canEditRuleWithActions(rowItem, actionsPrivileges),
+    onClick: async (rowItem: Rule) => {
       const createdRules = await duplicateRulesAction(
-        [extractRuleFromRow(rowItem)],
+        [rowItem],
         [rowItem.id],
```

Comment:
Looks like duplication of rule fails, because of additional introduced properties in sent rule body:  bulk_create_time_durations, search_after_time_durations.

Here is error response, when request is sent
POST /api/detection_engine/rules/_bulk_create
```JSON
{"statusCode":400,"error":"Bad Request","message":"[request body]: invalid keys \"bulk_create_time_durations,[],search_after_time_durations,[]\""}
```

Reply:
ah, I can see similar failure was already caught by tests https://github.com/elastic/kibana/pull/120217#issuecomment-985563389

Reply:
Nice catch, thank you 👍